frontend/public/components/service-instance.tsx

```diff
@@ -79,40 +79,58 @@ ServiceInstanceDetailsPage.displayName = 'ServiceInstanceDetailsPage';
 
 const ServiceInstancesHeader = props => <ListHeader>
   <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
-  <ColHead {...props} className="col-md-2 col-sm-4 hidden-xs" sortFunc="serviceCatalogStatus">Status</ColHead>
-  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortFunc="planExternalName">Plan</ColHead>
+  <ColHead {...props} className="col-md-2 col-sm-3 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-md-2 col-sm-3 hidden-xs" sortField="metadata.className">Service Class</ColHead>
```

Comment:
`metadata.className` does not exist, so the sort won't work. I would change this to

```
sortField="spec.clusterServiceClassExternalName"
```

although to handle namespaced service classes, we would need a custom `sortFunc`. (The above is OK for now.)